### ISSUE TITLE
Beaker Inventories deduplicating hosts

### ISSUE THREAD

**Issue by 14rcole** [CentOS-PaaS-SIG/linchpin#795]
We use the `system` key as `__IP__` for beaker.  That leads to the following error:

Given this res_output in linchpin.latest:
```json
"beaker_res": [
  {
    "status": "Running",
    "name": "rhel75-1",
    "family": "RedHatEnterpriseLinux7",
    "url": "https://beaker.engineering.redhat.com/jobs/3096027",
    "variant": "Server",
    "system": "hp-z400-01.rhts.eng.bos.redhat.com",
    "result": "Pass",
    "rid": "6119758",
    "arch": "x86_64",
    "id": "3096027",
    "distro": "RHEL-7.4"
  },  
  {
    "status": "Running",
    "name": "rhel75-1",
    "family": "RedHatEnterpriseLinux7",
    "url": "https://beaker.engineering.redhat.com/jobs/3096027",
    "variant": "Server",
    "system": "intel-canoepass-12.lab.bos.redhat.com",
    "result": "Pass",
    "rid": "6119759",
    "arch": "x86_64",
    "id": "3096027",
    "distro": "RHEL-7.5"
  },  
  {
    "status": "Running",
    "name": "rhel75-2",
    "family": "RedHatEnterpriseLinux7",
    "url": "https://beaker.engineering.redhat.com/jobs/3096027",
    "variant": "Server",
    "system": "hp-z400-01.rhts.eng.bos.redhat.com",
    "result": "Pass",
    "rid": "6119758",
    "arch": "x86_64",
    "id": "3096027",
    "distro": "RHEL-7.4"
  },  
  {
    "status": "Running",
    "name": "rhel75-2",
    "family": "RedHatEnterpriseLinux7",
    "url": "https://beaker.engineering.redhat.com/jobs/3096027",
    "variant": "Server",
    "system": "intel-canoepass-12.lab.bos.redhat.com",
    "result": "Pass",
    "rid": "6119759",
    "arch": "x86_64",
    "id": "3096027",
    "distro": "RHEL-7.5"
  }
],
```
We only output two beaker hosts to the inventory.  This is because, by default, we use the `system` field which, as you can see, is duplicated.  Two hosts have `hp-z400-01.rhts.eng.bos.redhat.com` and two have `intel-canoepass-12.lab.bos.redhat.com`.  To get truly unique hosts listed in the inventory, we need to set the hostname to something along the lines of  `__SYSTEM__-__NAME`.  `Name` is also duplicated, but each host has a unique system-name combination. The section of the inventory generated by this linchpin.latest output is below for reference:

```ini
[beaker-layout]
hp-z400-01.rhts.eng.bos.redhat.com hostname=hp-z400-01.rhts.eng.bos.redhat.com
intel-canoepass-12.lab.bos.redhat.com hostname=intel-canoepass-12.lab.bos.redhat.com
```

**Comment by p3ck** [comment]
There is something else going on..

https://beaker.engineering.redhat.com/jobs/3096027  Shows just two recipes/hosts.  It is not possible to have duplicate RUNNING hosts in beaker.

**Comment by p3ck** [comment]
I think it stems from some of the filtering done in linchpin/linchpin/provision/roles/beaker/tasks/provision_resource_group.yml

Look for these set_fact commands
  - name: "combine bkr_hosts and bkr_names"

I looked at this a while ago but was scared to change too much. :-)

**Comment by 14rcole** [comment]
Fixed in #800
